ec2/terraform/main.tf

```diff
@@ -55,7 +55,7 @@ resource "aws_instance" "ubuntu-ec2" {
   provisioner "remote-exec" {
     inline = [
       "cd /tmp/examples",
-      "git checkout ec2",
+      "git checkout bookinfo",
```

Comment:
Why are we doing a git checkout of bookinfo (or ec2)?   Presumably once this merges to master we'd use the code on the master branch (not on the bookinfo... or ec2... branch)?   Should this just use the code from whatever branch it is on?

Reply:
I see... because you're using ec2 to test the bookinfo changes and its doing its own clone you need it to grab this branch.    Ok, please be sure to do another commit after this to remove the checkout of the bookinfo branch.